tests/impls/image_reader/test_gdal_io.py

```diff
@@ -26,14 +25,14 @@
 from tests import TEST_DATA_DIR
 
 
-GH_IMAGE_FP = None              # type: Optional[str]
-GH_FILE_ELEMENT = None          # type: Optional[DataFileElement]
-GH_CROPPED_IMAGE_FP = None      # type: Optional[str]
-GH_CROPPED_FILE_ELEMENT = None  # type: Optional[DataFileElement]
-GH_CROPPED_BBOX = None          # type: Optional[AxisAlignedBoundingBox]
+GH_IMAGE_FP: str
+GH_FILE_ELEMENT: DataFileElement
+GH_CROPPED_IMAGE_FP: str
+GH_CROPPED_FILE_ELEMENT: DataFileElement
+GH_CROPPED_BBOX: AxisAlignedBoundingBox
```

Comment:
@Purg I'm getting flake8 errors that these variables are undefined. Is there something I'm missing?

Reply:
I suppose that makes some sense, they indeed are not defined. Did you try setting them to None without changing the type annotations? Did mypy not like that? We could move them into the classes as class-level attributes like in the PIL unittests.